src/modules/hype_chat/styles.module.css

```diff
@@ -0,0 +1,13 @@
+.hideHypeChatMessages {
+  div[class$='paid-pinned-chat-message-content-wrapper'] {
```

Comment:
can we scope this down further or match against the fully qualified class names?

Reply:
The styles are not applied to the body anymore, `.hideHypeChatMessages` is applied to the `.streamChat` and `.hideHypeChatButton` is applied to the chat box.